Assignment3/Dictionary.py

```diff
@@ -0,0 +1,21 @@
+# coding=utf-8
+class Dictionary:
+    """Dummy implementation of the Dictionary class for testing purposes.
+       In the real problem the methods isWord(string) and isPrefix(string) are assumed to have
+       θ(n) running time where n is the number of characters in the string."""
+
+    def __init__(self, word_list, prefix_list):
```

Comment:
Yes, this looks nice and makes it a lot easier (and less error prone) to create a dictionary.